### DISCUSSION TITLE
Ejercicio 2 - Tp8

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#207] [Preguntas y respuestas] [answered]
Hola, como va?
Consulto porque no entiendo bien como se tiene que interpretar la logica de la estructura para la fila:
```
typedef struct cola {
    nodo_t* frente;
    nodo_t* final;
} cola_t;
```
entiendo que el nodo "frente" seria como la primer persona en la fila el cual contiene un dato y seria el numero 1 supongo y que el nodo final es la actual persona al final de la fila. 

Mi pregunta es, tengo que ir creando mas nodos que le siguen al nodo "final"? o mantener el nodo final como el ultimo siempre ?

No se si se entiende mi duda. Gracias!

**Comment by martinvilu** [comment, marked as answer]
El primero que entra, es el primero que sale; igual que una fila en un supermercado, en algunos lados lo vas a encontrar como que el frente es el primero que entro. Esto es nomenclatura nomás. Lo que importa es el orden en el que entran y salen.

![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/b5aa7801-0782-440c-83b8-3de1365e0546)

Esta estructura, agrupa por donde empieza y por donde termina (lo que está en el medio, mucho no importa)
